### DISCUSSION TITLE
Just starting out

### DISCUSSION THREAD

**Discussion by outdoor720** [Zer0-bit/gaggiuino#554] [General]
if i purchase a Gaggia RI9380/47 Classic Pro Espresso.  Can i just purchase the peak coffee Gaggiuino V3 Kit set ( Gaggia Classic Pro ) kit and will it contain everything i need to mod the machine up to STM32 Blackpill features?  Does that include all possible mods?  Or are there other mods i should consider?  I live in Canada, so are there any issues with the kit and mods?

**Comment by Zer0-bit** [comment]
You need the peak kit + 3d prints.
